termux-auth.c

```diff
@@ -89,6 +89,14 @@ bool termux_change_passwd(const char *new_password) {
     return is_password_changed;
 }
 
+// Remove file that stores password hash
+// Return true on success, false otherwise.
+bool termux_remove_passwd(void) {
+    int n = remove(AUTH_HASH_FILE_PATH);
+
+    return n == 0;
```

Comment:
If .termux_authinfo does not exist, because it has not been set, then `remove`, and hence `termux-auth -l`, will return an error. 

It's not necessarily a problem, but might be more intuitive to not return an error in that particular case. Can be fixed by first checking if `AUTH_HASH_FILE_PATH` exists before trying to remove it

Reply:
Good idea